### PR TITLE
Fix scalebar with arbitrary label rotation.

### PR DESCRIPTION
**Pull request by pvalsecc**
Allow to have only one interval.

### CODE REVIEW

File: core/src/test/java/org/mapfish/print/processor/map/scalebar/ScalebarDrawerTest.java [hunk 1]
```diff
@@ -360,7 +360,7 @@ public void testDrawBarTopRight() throws Exception {
         ScalebarDrawer drawer = Type.BAR.createDrawer(graphics2d, settings);
         drawer.draw();
 
-//        ImageSimilarity.writeUncompressedImage(bufferedImage, "/tmp/expected-scalebar-top-right.tiff");
+        ImageSimilarity.writeUncompressedImage(bufferedImage, "/tmp/expected-scalebar-top-right.tiff");
         new ImageSimilarity(bufferedImage, 4).assertSimilarity(getFile(expectedDir + "expected-scalebar-top-right.tiff"), 44);
```
Comment (NielsCharlier) [inline core/src/test/java/org/mapfish/print/processor/map/scalebar/ScalebarDrawerTest.java]: I think you forgot to uncomment this one

Comment (pvalsecc) [inline core/src/test/java/org/mapfish/print/processor/map/scalebar/ScalebarDrawerTest.java]: Good catch. Fixed

File: core/src/main/java/org/mapfish/print/test/util/ImageSimilarity.java [hunk 2]
```diff
@@ -149,10 +151,20 @@ private double calcDistance(final BufferedImage other) {
      * @param maxDistance the maximum distance between the two images.
      */
     public void assertSimilarity(File other, double maxDistance) throws IOException {
+        final File actualOutput = new File(other.getParentFile(),
+                "actual" + other.getName().replace("expected", "").replace(".tiff", ".png"));
```
Comment (sbrunner) [inline core/src/main/java/org/mapfish/print/test/util/ImageSimilarity.java]: Why removing expected?

Comment (pvalsecc) [inline core/src/main/java/org/mapfish/print/test/util/ImageSimilarity.java]: To put actual at it's place if there was an expected...

Comment (sbrunner) [inline core/src/main/java/org/mapfish/print/test/util/ImageSimilarity.java]: ok :-)

**Pull request merged**